### DISCUSSION TITLE
Accordion - Align open text

### DISCUSSION THREAD

**Discussion by msagrilo** [Telefonica/mistica-design#1738] [Proposals]
### Goal

O objetivo é mudar o alinhamento do texto do accordion, quando ele está aberto.

### Context

Quando temos a versão grouped accordion, a hierarquia de leitura fica ruim.

### Impact

Trazer um conforto visual na leitura.

### Discussion Questions

Fiz uma mini poc para ajudar na percepção.

### Do you have a visual proposal?

https://www.figma.com/file/12wZyia8Y0vbx8fFpiP9Pm/Accordion-Evolution?type=design&node-id=2015%3A550&mode=design&t=BsB94RXFGdFilS5N-1

**Comment by yceballost** [comment]
Hi Marcelo! 

Nice proposal! The idea of adjusting the text alignment in the grouped accordion will definitely help improve visual clarity. We are interested in integrating these changes. We will to create an issue of this. I recommend you to generate a ticket in your Jira board to prioritize this issue from others :) 

#1755

**Comment by aweell** [comment]
Hi @msagrilo 

I reopen this discussion since after reviewing in detail the implications of the change, we're finding several issues.

I already mentioned some concerns in a comment on the issue (#1755) and after discussing it with the development team, although it is feasible, they do not consider it ideal.

In summary:

* To align it according to the need described in this discussion, it would be necessary to use JavaScript.
* With SSR (Server Side Rendering), it would not be possible to calculate the size of the asset on the server.
* If an accordion is open by default, it would need to be re-rendered to apply the asset size and align it correctly. This could prevent it from being displayed until that second render, but it would impact performance.

We really thought this was a good direction to evolve the accordion, as we already commented back in April, but given all the previous reasons we're inclined to advise against making this change.
